reflex-dom-core/src/Reflex/Dom/Builder/Immediate.hs

```diff
@@ -803,9 +808,13 @@ hydrateElement elementTag cfg child = do
               True -> go (Just node) -- this element should be skipped by hydration
               False -> do
                 t <- Element.getTagName e
-                -- TODO: check attributes?
                 if T.toCaseFold elementTag == T.toCaseFold t
-                  then pure e
+                  then do
+                    let shouldPatchAttrs = True -- TODO do something with this flag if hydration is much slower
```

Comment:
We did determine that it is much slower, right?

Reply:
Not yet, I abandoned performance stuff when we decided not to patch the attributes (and this code should no longer be here).